user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md

```diff
@@ -49,6 +49,10 @@ Expected duration: 20 minutes
 
 1. Click the *Deploy* button to deploy the DOM Editor package to your DMA.
 
+1. Go to <https://catalog.dataminer.services/details/package/5995>.
+
+1. Click the *Deploy* button to deploy the Kata Dom package to your DMA.
```

Comment:
This is already included under step 2 below. Is there a specific reason why you propose to already add this in step 1?

Reply:
I have completely missed step 2. I am sorry.